src/pages/developers/omnichain/tutorials/pyth.mdx

```diff
@@ -37,3 +37,26 @@ ensuring that the price feeds are accurate and up-to-date.
 Currently supported assets: BTC, BNB, MATIC, ETH, USDC, ZETA, USDT.
 
 Price feed IDs: https://pyth.network/developers/price-feed-ids
+
+# Pyth Entropy
+
+[Pyth Entropy] (https://docs.pyth.network/entropy) allows developers to quickly and easily generate secure 
+random numbers on the blockchain. Entropy's rapid response time allows 
+developers to build applications such as NFT mints and games with responsive UX. 
+Entropy also provides strong security guarantees to ensure that both 
+users and application developers can trust that the results are random.
+
+Pyth Entropy is currently available on Zetachain. [Learn more] (https://docs.pyth.network/entropy/generate-random-numbers) on how to generate a random number through Entropy.
```

Comment:
```suggestion
Pyth Entropy is currently available on Zetachain. [Learn more](https://docs.pyth.network/entropy/generate-random-numbers) on how to generate a random number through Entropy.
```

Just making sure links are valid, otherwise we get something like this:

<img width="851" alt="Screenshot 2024-05-21 at 20 54 35" src="https://github.com/zeta-chain/docs/assets/332151/9f5a03df-f259-48eb-9e6d-3ef08a9c923c">